src/components/breadcrumbs/README.md

```diff
@@ -332,39 +324,12 @@ If you are using Nunjucks,then macros take the following arguments
 
 </table>
 
-</div>
-
 ### Setting up Nunjucks views and paths
 
 Below is an example setup using express configure views:
 
-<pre>  `nunjucks.configure('node_modules/@govuk-frontend`, {
-    autoescape: true,
-    cache: false,
-    express: app
-  })` 
-  </pre>
-
-## Getting updates
```

Comment:
All of the READMEs appear to be missing everything from here to the license, despite most of it still being in the template?